_posts/2024-09-01-openloco-v24.09.md

```diff
@@ -0,0 +1,84 @@
+---
+title:      "OpenLoco version 24.09"
+tagline:    "Release blog post"
+author:     "Duncan Frost"
+date:       2024-09-01 21:30:00 +0100
+layout:     single
+classes:    wide
+categories: news
+---
+
+OpenLoco v24.09 is out! In this release we have made a few little tweaks to the
+landscape generation and implemented various vehicle functions.
+
+For a complete summary of changes, please find the
+[OpenLoco changelog](https://github.com/OpenLoco/OpenLoco/releases/tag/v24.09) on GitHub.
+
+## Implement Update Track / Road Motion (#2174, #2588)
+
+Vehicles travel along tracks and road in much the same way. There are fixed lanes that they can
+travel with all positions along the lane precomputed. When a road vehicle needs to overtake it
+enters/leaves an overtake lane. Moving along a lane is controled by the UpdateMotion family of
+functions. With these functions a speed is converted into a distance travelled in a single update
+tick and then it iteratively looks up the new positions in the precomputed lane data (sub position
+data). In OpenRCT2 these functions were very complex and handled a lot of special case situations
```

Comment:
grammatical weirdness here. I'd just add a `.` after " ...special cast situations" and start the next line with a new sentence.

Reply:
or just add an `and` at the end of the line